rllib/algorithms/appo/torch/appo_torch_learner.py

```diff
@@ -0,0 +1,187 @@
+from typing import Any, Dict, Mapping
+
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.algorithms.appo.appo_learner import (
+    AppoLearner,
+    LEARNER_RESULTS_CURR_KL_COEFF_KEY,
+    LEARNER_RESULTS_KL_KEY,
+    OLD_ACTION_DIST_KEY,
+)
+from ray.rllib.algorithms.impala.torch.vtrace_torch_v2 import (
+    make_time_major,
+    vtrace_torch,
+)
+from ray.rllib.core.learner.learner import POLICY_LOSS_KEY, VF_LOSS_KEY, ENTROPY_KEY
+from ray.rllib.core.learner.torch.torch_learner import TorchLearner
+from ray.rllib.core.rl_module.marl_module import ModuleID
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+
+
+class APPOTorchLearner(TorchLearner, AppoLearner):
+    """Implements APPO loss / update logic on top of ImpalaTorchLearner."""
+
+    def __init__(self, *args, **kwargs):
```

Comment:
I've personally never been a fan of when we pass args and kwargs here. makes it so that i have to hop between multiple files to know what args / kwargs to pass.